backend/dissemination/api/api_v1_1_0/drop.sql

```diff
@@ -1,7 +1,7 @@
 
 begin;
 
-DROP SCHEMA IF EXISTS api_v1_0_3 CASCADE;
+DROP SCHEMA IF EXISTS api_v1_1_0 CASCADE;
 -- DROP ROLE IF EXISTS authenticator;
```

Comment:
We can get rid of these, or add an explicit comment that we are not dropping these because doing so creates a cascade of problems for teardown and standup.